extra-dev/packages/coq-metalib/coq-metalib.dev/opam

```diff
@@ -0,0 +1,20 @@
+opam-version: "2.0"
+name: "coq-metalib"
+version: "dev"
+synopsis: "Locally Nameless Metatheory Library"
+maintainer: "Stephanie Weirich <sweirich@cis.upenn.edu>"
```

Comment:
This should be the maintainer of the package, not the maintainer of the software.

Reply:
OK I take the responsibility as maintainer.